CHANGELOG.md

```diff
@@ -1,5 +1,8 @@
 # Master
 
+* Add support for multi application setup
+* BREAKING CHANGE: Rename BookingSync::Engine::Model in BookingSync::Engine::AccountModel
```

Comment:
that will require 4.0 release then

Reply:
Actually, why do we need that that renaming?

Reply:
Exact, the renaming is optional yet due to the new ApplicationModel it feels much cleaner to adjust this.